src/components/Personalization/event.js

```diff
@@ -19,6 +19,11 @@ export const mergeDecisionsMeta = (
   eventType,
   eventLabel = ""
 ) => {
+  // Do not send a display notification with no decisions. Even empty view changes
+  // should include a proposition.
```

Comment:
let's move this logic where the event is triggered instead, this functions are calling `event` API

Reply:
Fixed in #1061